src/accounts/api/views.py

```diff
@@ -8,6 +8,8 @@
 from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
 from rest_framework.serializers import ValidationError
+from django.contrib.auth.models import User
```

Comment:
```suggestion
from .accounts.models import Account
```

use our Account class instead of the base User class.